wps-common/src/main/java/au/org/aodn/aws/wps/status/S3StatusUpdater.java

```diff
@@ -25,10 +26,26 @@ public S3StatusUpdater(String s3bucket, String uniqueId) {
     }
 
     public void updateStatus (String statusDocument) throws UnsupportedEncodingException {
-        AmazonS3 cl = AmazonS3ClientBuilder.defaultClient();
-        cl.putObject(
-                new PutObjectRequest(s3bucket, uniqueId + "/" + statusFilename, new StringInputStream(statusDocument), new ObjectMetadata())
-                        .withCannedAcl(CannedAccessControlList.PublicRead));
+        System.out.println("Updating status in S3 : " + statusDocument);
```

Comment:
perhaps use logger at some point so we can log different amounts of info in dev to prod

Reply:
I knew you'd say that  ;)